pom.xml

```diff
@@ -8,6 +8,76 @@
     <artifactId>candy-doc</artifactId>
     <version>0.0.1-SNAPSHOT</version>
     <packaging>maven-plugin</packaging>
+    <name>Candy Doc</name>
+    <description>Candy doc plugin</description>
+    <url>https://github.com/Candy-Doc/candy-doc-maven-plugin</url>
+
+    <licenses>
+        <license>
+            <name>GPL-3.0 License</name>
+            <url>https://www.gnu.org/licenses/gpl-3.0.en.html</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+        
+    
+    <developers>
+        <developer>
+            <name>Edouard Cattez</name>
+            <email>ecattez@ippon.fr</email>
+            <organization>Ippon</organization>
+            <organizationUrl>https://fr.ippon.tech/</organizationUrl>
+        </developer>
+        <developer>
+            <name>Maxime Deroullers</name>
+            <email>mderoullers@ippon.fr</email>
+            <organization>Ippon</organization>
+            <organizationUrl>https://fr.ippon.tech/</organizationUrl>
+        </developer>
+        <developer>
+            <name>Pierre Michiels</name>
+            <email>p.michiels@hotmail.com</email>
+            <organization>Ippon</organization>
+            <organizationUrl>https://fr.ippon.tech/</organizationUrl>
+        </developer>
+        <developer>
+            <name>Florian Louvet</name>
+            <email>flouvet@ippon.fr</email>
+            <organization>Ippon</organization>
+            <organizationUrl>https://fr.ippon.tech/</organizationUrl>
+        </developer>
+        <developer>
+            <name>Steeven Regnault</name>
+            <email>sregnault@ippon.fr</email>
+            <organization>Ippon</organization>
+            <organizationUrl>https://fr.ippon.tech/</organizationUrl>
+        </developer>
+        <developer>
+            <name>Charles Tacquet</name>
+            <email>ctacquet@ippon.fr</email>
+            <organization>Ippon</organization>
+            <organizationUrl>https://fr.ippon.tech/</organizationUrl>
+        </developer>
+    </developers>
+
+    <distributionManagement>
+        <snapshotRepository>
+            <id>ossrh</id>
+            <url>https://s01.oss.sonatype.org/content/repositories/snapshots</url>
```

Comment:
Check with @ctacquet if we succeed to pull image from maven central

Reply:
It works 🎉